conda_lock/conda_lock.py

```diff
@@ -1191,8 +1191,10 @@ def main() -> None:
     Literal["CRITICAL"],
 ]
 
+CONTEXT_SETTINGS = {"show_default": True, "help_option_names": ["--help", "-h"]}
```

Comment:
I did `["--help", "-h"]` instead of the more common `["-h", "--help"]` to keep the suggestion in the CLI `--help.`